docs/explanations/access_control.md

```diff
@@ -34,8 +34,8 @@ Synapse provides special principals for common sharing scenarios:
 
 - **273948**: All authenticated Synapse users
 - **273949**: Public access (anyone on the internet)
-- **Specific User ID**: Individual Synapse users (e.g., 3417048)
-- **Team ID**: Synapse teams for group-based permissions
+- **Specific User ID**: Individual Synapse users (e.g., #######)
+- **Team ID**: Synapse teams for group-based permissions (e.g., #######)
```

Comment:
is this `e.g., #######`? how about something like `syn123` so that it is more readable?

Reply:
Teams are integers, they do not have the `syn` prefix.